cmd/cdi-importer/importer.go

```diff
@@ -59,6 +60,16 @@ func waitForReadyFile() {
 	os.Exit(1)
 }
 
+func getHTTPEp(ep string) string {
+	readyFile, err := util.ParseEnvVar(common.ImporterReadyFile, false)
+	if err == nil && len(readyFile) > 0 {
+		if imageName, err := ioutil.ReadFile(readyFile); err == nil && len(imageName) > 0 {
```

Comment:
Same comment about err, but I get if we just want to return `ep` in that case, maybe only print it?

Reply:
fixed